src/shape/compound.rs

```diff
@@ -51,6 +52,58 @@ impl<N: RealField> Compound<N> {
             nbits,
         }
     }
+
+    /// Creates a new 2D concave polygon from a set of points assumed to describe a
+    /// counter-clockwise convex polyline.
+    pub fn new_concave_polygon(isometry: Isometry<N>, hull: &[(N, N)]) -> Compound<N>
```

Comment:
I don't know if `new_concave_polygon` should be added as a new constructor of `Compound` or if a new struct that implements the same traits as `Compound` should be created.

The issue with the current proposal is that the triangles used for the creation of the `Compound` shape are leaked, since it's possible to manipulate them with `Compound::shapes()` and then cast them with `.as_shape::<ConvexPolygon<f64>>()`.